zip-0208.rst

```diff
@@ -0,0 +1,224 @@
+::
+
+  ZIP: 208
+  Title: Shorter Block Target Spacing
+  Author: Simon Liu <simon@z.cash>
+          Daira Hopwood <daira@z.cash>
+  Category: Consensus
+  Created: 2018-01-10
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST", "SHOULD", "SHOULD NOT", and "MAY" in this document are to
+be interpreted as described in RFC 2119. [#RFC2119]_
+
+The terms "block chain", "consensus rule change", "branch", and "network
+upgrade" are to be interpreted as defined in [#zip-0200]_.
+
+The term "block target spacing" means the time interval between blocks targeted
+by the difficulty adjustment algorithm in a given branch. It is normally
+measured in seconds. (This is also sometimes called the "target block time",
+but "block target spacing" is the term used in [#latest-protocol]_.)
+
+
+Abstract
+========
+
+This proposal specifies a change in the block target spacing, to take effect in
+the Blossom network upgrade [#zip-0206]_.
+
+The emission schedule of mined ZEC will be approximately the same in terms of
+time, but this requires the emission per block to be adjusted to take account
+of the changed block target spacing.
+
+
+Motivation
+==========
+
+The motivations for decreasing the block target spacing are:
+
+- Reduced latency for considering transactions to be sufficiently confirmed;
+- Greater throughput of transactions in unit time.
+
+The latter goal could alternatively be achieved by increasing the block size
+limit, but that would not also achieve the former goal.
+
+Note that, for a given security requirement (in terms of the expected cost
+distribution of a rollback attack), the number of confirmations needed
+increases more slowly than the decrease in block time, and so, up to a point,
+decreasing the block target spacing can provide a better trade-off between
+latency and security. This argument assumes that the verification and
+propagation time for a block remain small compared to the block target spacing.
+
+
+Specification
+=============
+
+The following constants are defined:
+
+- PreBlossomPoWTargetSpacing := 150 seconds
+- PostBlossomPoWTargetSpacing := 75 seconds
+
+TODO: check that PostBlossomPoWTargetSpacing is reasonable.
+
+For a given network (production or test), we define IsBlossomActivated(height)
+to return true if Blossom has activated at the given block height, otherwise
+false. The activation heights for production and test networks are defined in
+[#zip-0206]_.
+
+This specification assumes that IsBlossomActivated(height) returns false for
+height < SlowStartInterval.
+
+Define BlossomActivationHeight to be the first height at which
+IsBlossomActivated(height) returns true for this network.
+
+Define:
+
+- BlossomPoWTargetSpacingRatio := PreBlossomPoWTargetSpacing / PostBlossomPoWTargetSpacing
```

Comment:
The current upper "Is It Worth It" bound I establish in this review implies `BlossomPoWTargetSpacingRatio ≥ 2` otherwise the complexity isn't worth it.